src/columns/session.rs

```diff
@@ -40,3 +42,19 @@ impl Column for Session {
 
     column_default!(i32);
 }
+
+#[cfg(target_os = "macos")]
+impl Column for Session {
+    fn add(&mut self, proc: &ProcessInfo) {
+        let sid = unistd::getsid(Some(Pid::from_raw(proc.pid)))
```

Comment:
Is this actually different from how it's done in linux/BSD ?

Reply:
In Linux, session can be got by parsing `/proc/[PID]/stat`.
This is lightweight than calling `getsid` by each proccess.